_config/logging.yml

```diff
@@ -17,7 +17,7 @@ SilverStripe\Core\Injector\Injector:
     constructor:
       - "error-log"
     calls:
-      DisplayErrorHandler: [ pushHandler, [ %$Monolog\Handler\HandlerInterface ] ]
+      pushDisplayErrorHandler: [ pushHandler, [ %$Monolog\Handler\HandlerInterface ] ]
```

Comment:
I'd like to see the docs updated for this as well, as it's an unclear part of the API :D